ci-operator/config/openshift/loki/openshift-loki-release-5.8.yaml

```
@@ -0,0 +1,153 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
```

I think we're missing the `loki` component:

```suggestion
    tag: base
  loki:
    name: v2.9.1
    namespace: logging
    tag: loki
```